frontend/src/app/main/ui/icons.clj

```diff
@@ -11,9 +11,9 @@
    [rumext.v2]))
 
 (defmacro icon-xref
-  [id]
+  [id & class]
```

Comment:
This class is a seq of the optional arguments, si it is not that you are expecting

Reply:
I guess it kinda worked before because of the `~`? O_O (not sure what operator does) I think I fixed it now, thanks!